packages/client/src/views/AdvancedSearch/SaveBookmarkModal.tsx

```diff
@@ -59,11 +59,13 @@ export function BookmarkAdvancedSearchModal({
   const { searchId, ...advancedSearchParams } = useSelector(
     getAdvancedSearchParamsState
   )
+  delete advancedSearchParams.bookmarkName
```

Comment:
Why is it that we need to remove the `bookmarkName`?

Reply:
gateway type [AdvancedSeachParameters](https://github.com/opencrvs/opencrvs-core/blob/5986f91185cdd9e9dc05f219b62dee4d2473f521/packages/client/src/utils/gateway.ts#L105-L106) does not have `bookmarkName`

Reply:
You mean [AdvancedSeachParametersInput](https://github.com/opencrvs/opencrvs-core/blob/5986f91185cdd9e9dc05f219b62dee4d2473f521/packages/client/src/utils/gateway.ts#L166) instead?